lib/cloudkeeper/aws/cloud.rb

```diff
@@ -49,6 +51,7 @@ def delete_data(file_name)
       # @param appliance [Appliance] data about image
       # @return [Number] import task id
       def start_import_image(appliance)
+        logger.debug { "Starting import image task for #{appliance}" }
```

Comment:
I think this will print useless object description for appliance instead of its identifier.